src/index.js

```diff
@@ -80,8 +101,56 @@ const displayUserTrips = (traveler) => {
   })
 }
 
+const findDuration = (start, end) => {
+  let startDate = new Date(start);
+  let endDate = new Date(end);
+  return (endDate.getDate()) - (startDate.getDate())
+}
+
+const displayEstimate = (newTrip, destinationData) => {
+  tripPreviewTitle.innerText = `Your trip to ${destinationData.name}`
+  tripPreviewData.innerHTML = `
+  <p class="destination">Destination ${destinationData.name}</p>
+  <p class="leaving">Departing on: ${newTrip.duration}</p>
+  <p class="returning">Returning on: ${newTrip.duration}</p>
+  <p class="duration">Duration ${newTrip.duration}</p>
```

Comment:
"Departing on" and "Returning on" items show a single whole number instead of a date. Is that the intention?